deployment/build_and_release/modules/presubmit_cloudbuild_triggers/variables.tf

```diff
@@ -90,4 +90,9 @@ variable "firmware_base_url" {
 variable "rest_distributor_base_url" {
   description = "Base URL for the checkpoint distributor"
   type = string
+}
+
+variable "bastion_addr" {
+  description = "Host:port of the bastion server"
+  type = string
```

Comment:
How about adding `default = ""` ? Similarly in the other `variables.tf`. As it stands, this doesn't modify the terragrunt files so I imagine this fails because of a missing required input variable. But its's been a while since I thought about this stuff, so maybe I'm off base. If you've tested and you're happy, ignore.